### DISCUSSION TITLE
open PR to make bio command fields inline

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#739] [First-Timers]
Original issue: [EddieBot#386](https://github.com/EddieJaoudeCommunity/EddieBot/issues/386) (created by @stemount)

Q) How to get started with this issue?

Have a look at the [src/commandHandlers/bio.ts](https://github.com/EddieJaoudeCommunity/EddieBot/blob/develop/src/commandHandlers/bio.ts) file.

Have a look at the argument called `inline` in the embed constructor.
https://discordjs.guide/popular-topics/embeds.html#using-the-richembedmessageembed-constructor

You could find all about making discord embeds here:
https://discordjs.guide/popular-topics/embeds.html

The current embeds look like this:
![](https://user-images.githubusercontent.com/150512/102241584-08c37600-3ef1-11eb-8399-35596d8dc5a8.png)

We could perhaps have them look like this->
![](https://user-images.githubusercontent.com/150512/102243115-b7b48180-3ef2-11eb-8bda-ecc9481bf9d4.png)

**Comment by eddiejaoude** [comment]
Thanks @Vyvy-vi 👍

**Comment by schmelto** [comment]
One other thought on this.
![Screenshot 2021-01-31 151309](https://user-images.githubusercontent.com/30869493/106386771-e5885380-63d6-11eb-8166-aede4708f8c2.jpg)
I think the reading bio is not necessary here and takes up a lot of unnecessary space :)

**Reply by eddiejaoude** [reply]
I think you are right, the `description` could go there 👍
